src/setupTests.js

```diff
@@ -7,4 +7,10 @@ import '@testing-library/jest-dom/extend-expect';
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
+//Required for plotly tests
```

Comment:
Sorry to nitpick but please use a space before the start of the comment so it's a little easier to skim read